### PR TITLE
Use view.bounds instead of UIApplicationDelegate.window.bounds

### PR DESCRIPTION
**Pull request by andreyrd**
We can't do `configureLayout(_ size: CGSize = view.bounds.size)` because instance members can't be used as default values, so as one of the possible workarounds, I just have the caller pass in the size it wants.

Edit: This is for #150

### CODE REVIEW

File: Source/LightboxController.swift [hunk 1]
```diff
@@ -271,7 +271,7 @@ open class LightboxController: UIViewController {
 
   // MARK: - Layout
 
-  open func configureLayout(_ size: CGSize = UIApplication.shared.delegate?.window??.bounds.size ?? .zero) {
+  open func configureLayout(_ size: CGSize) {
     scrollView.frame.size = size
```
Comment (onmyway133) [inline Source/LightboxController.swift]: We always pass `self.view.bounds.size` into `configureLayout`, then why do we need this parameter 🤔 @vadymmarkov @zenangst

Comment (vadymmarkov) [reply Source/LightboxController.swift]: I'm not sure why we have it, so I'm fine with removing this default value.

Comment (onmyway133) [reply Source/LightboxController.swift]: @andreyrd I think we can just use the default view size inside `configureLayout` without passing it as a parameter, mind making the change 😉

Comment (andreyrd) [reply Source/LightboxController.swift]: It looks like a value other than `view.bounds.size` is passed into `configureLayout` in `viewWillTransition`. (Should be line 220)

**Comment by onmyway133** [comment]
@andreyrd Many thanks for the PR, you're great 👍 We will inspect the `configureLayout`

**Pull request merged**